proto/buf/alpha/audit/v1alpha1/envelope.proto

```diff
@@ -614,6 +615,10 @@ message UserActor {
   string subject = 5;
 }
 
+message MachineActor {
+  string service_name = 1;
```

Comment:
Can you list the cases in which we'd start using service_name here? If a user triggers something on goproxyd and that reaches out to bufd, that would still be a user actor right?

Reply:
anything user triggered would still be user actor but for something like auto provisioning calling `AddUserToOrganization` this would allow us to use `envoyauthd` as the `service_name` (it's recorded as `anonymous` right now)